src/main/kotlin/com/cognifide/gradle/aem/environment/docker/base/DockerStack.kt

```diff
@@ -1,15 +1,21 @@
 package com.cognifide.gradle.aem.environment.docker.base
 
-open class DockerStack(val name: String) {
+import com.cognifide.gradle.aem.common.AemExtension
 
-    var initTimeout = 10000L
+open class DockerStack(private val aem: AemExtension, val name: String) {
 
-    var runningTimeout = 10000L
+    var initTimeout = aem.props.long("environment.dockerStack.initTimeout") ?: 10000L
+
+    var runningTimeout = aem.props.long("environment.dockerStack.runningTimeout") ?: 10000L
 
     fun init() {
         val result = Docker.execQuietly {
             withTimeoutMillis(initTimeout)
             withArgs("swarm", "init")
+
+            if (aem.environment.dockerType == DockerType.TOOLBOX) {
```

Comment:
this will handle a need to specify extra parameter to swarm especially when running on Docker Toolbox